Makefile

```
@@ -147,7 +147,7 @@ $(ENVTEST): $(LOCALBIN)
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
 img-test:
-	docker run --rm  -v $(SRCROOT):/luigi -w /luigi golang:1.17.7-bullseye  bash -c "make test"
+	docker run --rm  -v $(SRCROOT):/luigi -w /luigi golang:1.19.1-bullseye  bash -c "GOFLAGS=-buildvcs=false make test"
```

```suggestion
	docker run --rm  -v $(SRCROOT):/luigi -w /luigi golang:1.19-alpine3.18  bash -c "GOFLAGS=-buildvcs=false make test"
```
Alpine images are available - and `1.19` is equivalent to `1.19.13`, the latest patch for 1.19

---

controller-gen need additional tools that are not preinstalled in alpine (tried didn't work)